app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= stylesheet_link_tag 'session', media: 'all', 'data-turbolinks-track': 'reload' %>
+
+<div class="login-container">
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name), html: { class: "login-form" }) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "input-field" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", class: "input-field" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class:"login" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
+</div>
```

Comment:
- [ ] [REQUIRED] As I have recommended for the signup page, let's also adjust the width of the input fields and the Login button so that they align as designed in the wireframe provided.

### Your Design
<img width="676" alt="Screenshot 2023-09-07 at 22 18 25" src="https://github.com/AleWaweru/blog-App/assets/58692590/a7bc9df3-dc5d-428e-a53f-b1aac24c7f7b">

Reply:
done